source/about/mattermost-v10-changelog.md

```diff
@@ -7,17 +7,95 @@ Support for Mattermost Server v9.5 [Extended Support Release](https://docs.matte
+### Important Upgrade Notes
+ - We no longer support new installations using MySQL starting in v10. All new customers and/or deployments will only be supported with the minimum supported version of the PostgreSQL database. End of support for MySQL is targeted for Mattermost v11.
+ - Apps Framework is deprecated for new installs. Please extend Mattermost using webhooks, slash commands, OAuth2 apps, and plugins.
+ - An Enterprise license will be required to use v2.0+ Playbooks functionality starting in v10. Team Edition and Professional deployments can continue to use Playbooks v1.x. Feature updates will only be added to Playbooks v2.0+.
+ - Renamed ``Channel Moderation`` to ``Advanced Access Control`` in the channel management section in the **System Console**.
+ - Renamed announcement banner feature to “system-wide notifications”.
+ - Renamed “Collapsed Reply Threads” to “Threaded Discussions” in the System Console.
+ - Renamed “System Roles” to “Delegated Granular Administration” in the System Console.
+ - Fully deprecated the ``/api/v4/image`` endpoint when the image proxy is disabled.
+ - Pre-packaged Calls plugin [v1.0.0](https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v1.0.0). This includes breaking changes such as allowing calls in direct message channels only on unlicensed servers.
+ - Removed deprecated ``Config.ProductSettings``, ``LdapSettings.Trace``, ``AdvancedLoggingConfig``, and ``pageSize`` query parameter.
 
-### Upcoming Breaking Changes in v10.0 Release
+```{Important}
+If you upgrade from a release earlier than v9.11, please read the other [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html).
+```
+
+### Improvements
+
+#### User Interface (UI)
+ - Pre-packaged Calls plugin [v1.0.0](https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v1.0.0).
+ - Pre-packaged the MS Teams plugin for Mattermost, [v2.0.3](https://github.com/mattermost/mattermost-plugin-msteams/releases/tag/v2.0.3).
+ - Added Playbooks [v2.0.0](https://github.com/mattermost/mattermost-plugin-playbooks/releases/tag/v2.0.0) to the prepackaged plugins.
+ - Changed the right-hand side scroll direction and fixed the advanced text editor to the bottom.
+ - Added Do not disturb and late timezone warnings to Direct Message posts.
+ - Added user statuses to the Group Members modal.
+ - Added labels for channel header and purpose in the right-hand side channel info view.
+ - Added pagination user interface to the ``BackstageList`` component.
+ - Made various improvements to code involving user preferences.
+ - Promoted GIF picker, custom groups and message priority out of Beta.
+ - Removed the **Pre-release features** section from **Settings > Advanced** due to lack of usage.
+
+#### Administration
+ - Added experimental support for Redis. This is not intended for production use yet.
```

Comment:
Let's remove this. I don't want to confuse customers yet. We will announce when we are ready to announce.